tst/plistqueue.tst

```diff
@@ -0,0 +1,118 @@
+gap> q := PlistQueue(1000);
+<queue with 0/1000 entries>
+gap> q := PlistQueue(1000, "mist");
```

Comment:
So is that english or german "mist"? Perhaps pick a more neutral word, like "test"? :-)